backend/src/types.ts

```diff
@@ -91,6 +94,14 @@ export interface IImage {
     note: string;
 }
 
+
+export enum ReadState {
```

Comment:
please use a **union type** instead of an enum
see: https://fettblog.eu/tidy-typescript-avoid-enums/